util/test/Test/Pos/Util/Tripping.hs

```diff
@@ -85,14 +117,19 @@ trippingBuildable x enc dec =
                     failWith
                         (Just $ Diff "━━━ " "- Original" "/" "+ Roundtrip" " ━━━" diff) $
                             Prelude.unlines
-                            [ "━━━ Intermediate ━━━"
+                            [ mconcat ["━━━ ", format," ━━━"]
```

Comment:
`format` allows us to differentiate between `YAML` and `JSON` when failures occur.